lib/utils/index.js

```diff
@@ -2208,6 +2208,16 @@ module.exports = {
       (token, i) =>
         token.type === tokensR[i].type && token.value === tokensR[i].value
     )
+  },
+
+  /**
+   * TODO: replace usages with 'array.at(-1)' once the minimum supported Node.js version is at least 16.6.0
+   * @template T
+   * @param array {T[]}
+   * @returns {T | undefined}
+   * */
+  getLastArrayElement(array) {
```

Comment:
This is not necessary, just write it inline.
We can fix that later with a prefer-at rule.
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-at.md

Reply:
done ✅